### PR TITLE
he/she -> they

### PR DESCRIPTION
**Pull request by lxjv**
It's more inclusive and rolls off the tongue better

### CODE REVIEW

File: pages/02.applications/02.docs/funkwhale/app_funkwhale.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ routes:
 - [Limitations with YunoHost](#limitations-with-yunohost)
 - [Useful links](#useful-links)
 
-Funkwhale is a self-hosted, federated free music streaming server. Each utilisateur⋅trice has a library in which he/she can add the titles he/she wants, and which he/she can share with other users, whether they are on the same instance or not. [¹](#sources)
+Funkwhale is a self-hosted, federated free music streaming server. Each utilisateur⋅trice has a library in which they can add the titles they want, and which they can share with other users, whether they are on the same instance or not. [¹](#sources)
```
Comment (alexAubin) [inline pages/02.applications/02.docs/funkwhale/app_funkwhale.md:20]:
```suggestion
Funkwhale is a self-hosted, federated free music streaming server. Each users has a library in which they can add the titles they want, and which they can share with other users, whether they are on the same instance or not. [¹](#sources)
```

**Review by ericgaspar — APPROVED** [review]

**Review by alexAubin — COMMENTED** [review]
Indeed, cheers !

I guess we can also change the `utilisateur⋅trice` to just `users`, it's probably some sort of copypasta from the french version ?

**Comment by lxjv** [comment]
yeah i agree, i dont speak french and i didnt want to mess it up with a translation that wasnt up to par :)

**Comment by alexAubin** [comment]
Alrighty, cheers ! Fixed it in https://github.com/YunoHost/doc/commit/6fb3b4d819f3edf5f408aab7ae55d339c8c3200a :wink:

**Comment by alexAubin** [comment]
Ah :sweat_smile:

**Pull request merged**